src/vs/workbench/common/editor/editorGroupModel.ts

```diff
@@ -25,7 +25,7 @@ export interface IEditorOpenOptions {
 	readonly sticky?: boolean;
 	readonly transient?: boolean;
 	active?: boolean;
-	readonly selected?: boolean;
+	readonly addToSelection?: boolean;
```

Comment:
Maybe this should be `true | undefined` so that it is never `false`. Because otherwise it conflicts with `active: true`

Reply:
I think you misunderstand this. The main reason for having this is to handle `active: true` case correctly. Either the new active editor should become the only selected editor (default case) or the new active editor should become selected together with the already exiting selection.